### PR TITLE
Switch to the offical Appium NuGet package

### PR DESCRIPTION
**Pull request by karanbirsingh**
#### Describe the change
In the UI tests project, we were using a prerelease/temp version of the package from the WinAppDriver team. This code should be pulled from the official Appium package. This PR switches packages. This led to updates of Newtonsoft across the project because Appium requires >=12 and most of our code used 10.0.3. The versions were consolidated to >=12

#### PR checklist

- (saving/loading) Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- (no) Does this address an existing issue? If yes, Issue# - 
- (no) Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: src/AccessibilityInsights/MainWindow.xaml.cs [hunk 1]
```diff
@@ -159,6 +159,9 @@ public MainWindow()
 
             SetHighContrastTheme();
 
+            ///in case we need to do any debugging with elevated app
+            SupportDebugging();
```
Comment (jalkire) [inline src/AccessibilityInsights/MainWindow.xaml.cs]: Why was this moved? Curious

Comment (karanbirsingh) [reply src/AccessibilityInsights/MainWindow.xaml.cs]: I sometimes hit errors in `PopulateConfigurations` that occurred too early. This way we can step through PopulateConfigurations
